packages/dds/map/src/mapKernel.ts

```diff
@@ -32,7 +32,8 @@ interface IMapMessageHandler {
         op: IMapOperation,
         local: boolean,
         localOpMetadata: unknown,
-    ): void;
+        isStashedOp: boolean,
```

Comment:
I think it would be cleaner to have a new method for applying stashed ops rather than overloading the existing process method. With the overloading, we have somewhat redundant parameters that must be consistent (e.g. if isStashedOp === true, then local must be true and localOpMetadata must be undefined) and return value of localOpMetadata if isStashedOp is true. I don't think there would be much duplicated code (which could be pulled out into separate methods anyway) and there would be fewer if statements.

Reply:
Long story short, the question is regarding which approach is more desirable:
1. Processing stashed ops reusing normal local processing method. Pros: we make sure we are processing the stashed ops exactly like normal ops. Cons: there could be a couple ifs remaining even after refactoring mapKernel
2. Create a separate method to process stashedOps: Pros: Cleaner, no need to add extra ifs to current process method. Cons: repeated code, we could forget adding regular processing logic to stashed op logic.

Reply:
I agree with Scarlett. I don't think there's much duplicated code, but it could just be split out to a separate method.

It's also somewhat conceptually confusing since we use `process` in our codebase to mean handling an op that has been sequenced, which is not accurate for applyStashedOp() (in fact, there is no overlap between applyStashedOp() and processing a local op, since these are complementary).

Having a separate method for getting local op metadata that is called in applyStashedOp() and at submit time is a good change, though.